server/models/Collection.js

```diff
@@ -223,6 +226,17 @@ Collection.addHook("afterCreate", (model: Collection, options) => {
 
 // Class methods
 
+Collection.findByPk = async function (id, options = {}) {
+  if (isUUID(id)) {
+    return this.findOne({ where: { id }, ...options });
```

Comment:
We are using `findByPk` with two `scopes`. So I used `this` here. All the server test cases are passing, so I think `this` is using the scope returned by the `scopes`.

Reply:
This makes sense, a couple of explicit tests for the cases of passing a uuid / urlId here  would be good.

Reply:
Agree, Will do that.